haystack/nodes/prompt/invocation_layer/anthropic_claude.py

```diff
@@ -15,7 +15,7 @@
     AnthropicTokenStreamingHandler,
     DefaultTokenStreamingHandler,
 )
-from haystack.utils.requests import request_with_retry
+from haystack.utils.requests_utils import request_with_retry
```

Comment:
Can we import the name in the package init so we can have `from haystack.utils import request_with_retry`